ng/src/web/components/dialog/linkconfirmationdialog.js

```diff
@@ -0,0 +1,178 @@
+/* Greenbone Security Assistant
+ *
+ * Authors:
+ * Björn Ricks <bjoern.ricks@greenbone.net>
+ * Steffen Waterkamp <steffen.waterkamp@greenbone.net>
+ *
+ * Copyright:
+ * Copyright (C) 2017 Greenbone Networks GmbH
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import React from 'react';
+import glamorous from 'glamorous';
+
+import _ from 'gmp/locale.js';
+
+import {is_defined} from 'gmp/utils.js';
+
+import PropTypes from '../../utils/proptypes.js';
+
+import Dialog from '../dialog/dialog.js';
+import DialogContent from '../dialog/content.js';
+import DialogTitle from '../dialog/title.js';
+import ScrollableContent from '../dialog/scrollablecontent.js';
+
+import Button from '../dialog/button.js';
+
+import Layout from '../../components/layout/layout.js';
+
+const StyledLayout = glamorous(Layout)({
+  display: 'flex',
+  justifyContent: 'space-between',
+  borderWidth: '1px 0 0 0',
+  borderStyle: 'solid',
+  borderColor: '#ddd',
+  marginTop: '15px',
+  padding: '10px 15px 10px 15px',
+});
+
+class ConfirmationDialogContent extends React.Component {
+
+  constructor(...args) {
+    super(...args);
+
+    this.handleResume = this.handleResume.bind(this);
+    this.handleClose = this.handleClose.bind(this);
+  }
+
+  handleResume() {
+    const {
+      onResumeClick,
+      to,
+    } = this.props;
+
+    if (onResumeClick) {
+      const promise = onResumeClick(to);
+      if (is_defined(promise)) {
+        promise.then(
+          () => this.handleClose()
+        );
+      }
+      else {
+        this.handleClose();
+      }
+    }
+  }
+
+  handleClose() {
+    const {close} = this.props;
+    close();
+  }
+
+  render() {
+    let {
+      moveprops,
+      text,
+      title,
+    } = this.props;
+
+    // display slashes correctly, which have been escaped before
+    text = text.replace(/&#x2F;/g, '/');
```

Comment:
Now I understand why you are adding this :-) It's not necessary. I'll point you to the code how to fix it in another comment.